src/models/data.py

```diff
@@ -230,15 +230,20 @@ def ds_folder_to_np(self,
         yearly_mean = x.mean(dim=['time', 'lat', 'lon'])
         yearly_mean_np = yearly_mean.to_array().values
 
+        yearly_std = x.std(dim=['time', 'lat', 'lon'])
+        yearly_std_np = yearly_std.to_array().values
+
+        yearly_agg = np.concatenate((yearly_mean_np, yearly_std_np))
```

Comment:
what are the dimensions of this array then?

Reply:
`yearly_agg.shape == (num_variables * 2,)`